pages/mesosphere/dcos/2.2/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -124,11 +118,86 @@ If `ca_certificate_path` points to a self-signed root certificate then omit this
 
 Otherwise `ca_certificate_chain_path` points to a file containing the complete chain of certificates up to a self-signed root certificate, but excluding the signing certificate contained in `ca_certificate_path`. The order is significant: the first certificate must verify the signing certificate in `ca_certificate_path` and each subsequent certificate must verify the preceding certificate.
 
-## Install or patch DC/OS
+### ca\_truststore\_path
+Path (relative to the `$DCOS_INSTALL_DIR`) to a file containing additional trusted CA certificates. This should be a file containing one or more PEM certificates. For example: `genconf/dcos-truststore.pem`.
+
+The signing certificate root CA is automatically trusted and does not need to be added here.
+
+
+# Installing a cluster with a custom CA Certificate
+
+Create the signing files [as described above](#create-signing-ca-files) and copy them to the `$DCOS_INSTALL_DIR/genconf/` directory on the bootstrap node.
+
+Modify the DC/OS Configuration file [as described above](#dcos-configuration) to point to the signing files.
+
+Proceed with the installation by running the appropriate `dcos_generate_config.ee.sh` command as described in the [documentation of the Advanced Installer](/mesosphere/dcos/2.2/installing/production/deploying-dcos/installation). Note that the current working directory when executing `dcos_generate_config.ee.sh` must be the `$DCOS_INSTALL_DIR` directory.
+
+For security reasons, the installer will not copy the signing key file from the bootstrap node to the master nodes.
+Manually distribute the signing key to every DC/OS master node before running the `dcos-install.sh` script.
+If you copy the signing key file over the network onto the master nodes, the network channel must be adequately protected.
+
+On each master node:
+1. Copy the `dcos-install.sh` script to the node.
+2. Create the directory `/var/lib/dcos/pki/tls/CA/private/` owned by the `root` user and with permission mode 0700.
+3. Copy the signing key file to `/var/lib/dcos/pki/tls/CA/private/custom_ca.key`. Ensure that the signing key file is owned by the `root` user and is only accessible to the owner (permission mode 0600).
+4. Run the `dcos-install.sh` script as described in the installation instructions.
+
+# Changing the CA Certificate in an existing cluster
+
+To replace the CA certificate (for example, if it is expiring or compromised), perform the following procedure consisting of three patch upgrades.  This procedure ensures that the cluster stays healthy during the certificate replacement.  If Calico is enabled (the default setting), then workloads may be stopped.
```

Comment:
The first time I read this, I breezed over the 'three patch' part.  Repetition seems good here:

```suggestion
To replace the CA certificate (for example, if it is expiring or compromised), perform the following procedure consisting of three patch upgrades.  Three patch upgrades are required to ensure that the cluster stays healthy during the certificate replacement.  If Calico is enabled (the default setting), then workloads may be stopped.
```